controllers/etcdcluster_controller.go

```diff
@@ -277,6 +285,24 @@ func clientCertForStorageOSCluster(cluster *etcdv1alpha1.EtcdCluster, caCert, ca
 	}, nil
 }
 
+func (r *EtcdClusterReconciler) getTLSConfig(ctx context.Context, cluster *etcdv1alpha1.EtcdCluster) (*cryptotls.Config, error) {
+	var tlsConfig *cryptotls.Config
+	if cluster.Spec.TLS != nil && cluster.Spec.TLS.Enabled {
```

Comment:
Could you please swap the expression and do an early return, it should make code easier to read